packages/file-adapters/lib/local-file.js

```diff
@@ -3,9 +3,14 @@ const path = require('path');
 const mkdirp = require('mkdirp');
 
 module.exports = class LocalFileAdapter {
-  constructor({ src, path: inputPath }) {
+  constructor({ src, path: inputPath, getFilename: nameGetter }) {
     this.src = path.resolve(src);
     this.path = inputPath;
+    this.getFilename = nameGetter;
+
+    if (!this.getFilename) {
+      this.getFilename = this.getDefaultFilename;
```

Comment:
Can we use an inline function definition here:
```
this.getFileName = ({ id, originalFileName }) => `${id}-${originalFilename}`;
```

This saves us from adding the `getDefaultFilename` which increases the size of an public API we need to support.

Reply:
Done.